Disable "fp" tests since they live in a subrepo and are not fetched during
"make fetch"

--- tests/fp/meson.build.orig	2021-05-11 21:34:33 UTC
+++ tests/fp/meson.build
@@ -1,8 +1,4 @@
-# There are namespace pollution issues on Windows, due to osdep.h
-# bringing in Windows headers that define a FLOAT128 type.
-if targetos == 'windows'
-  subdir_done()
-endif
+subdir_done()

 fpcflags = [
   # softfloat defines
